app/pages/list/list.js

```diff
@@ -0,0 +1,34 @@
+import {IonicApp, Page, NavController, NavParams} from 'ionic/ionic';
+
+@Page({
+  templateUrl: 'build/pages/list/list.html'
+})
+export class ListPage {
```

Comment:
Here we prefix the class with export which we did not do with our root component. This will allow our component to be imported elsewhere, but since we don’t need to import our root component anywhere we don’t add it.